frontend/components/Social.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, BoxProps, Flex } from "@chakra-ui/react";
+import Link from "next/link";
+import React from "react";
+import Splash from "./Splash";
+
+export type SocialProps = React.PropsWithChildren<{
+  link: string;
+  color: string;
+  isBubbleActive: boolean;
+  onHoverColor: string;
+}>;
+
+export function Social({
+  link,
+  isBubbleActive,
+  color,
+  onHoverColor,
+  children,
+}: SocialProps) {
+  return (
+    <Link href={link} passHref>
+      <Box as="a" m="1">
+        {isBubbleActive ? (
```

Comment:
Do we actually need to render social icons without the bubble underneath?

Reply:
I've done the same thing as the hero section, I think it isn't a bad feature

Reply:
@BaggioGiacomo But we have to think in practice what's best.. We're not gonna do something just because it seems cool to do it.. Do actually content editors need to show social link without bubble? If not, why giving them the setting? Otherwise we might also enable the editor to change the color of the icon, or its size, just in case... But all those additional features will just add noise because they're not really needed.

Reply:
Yeah, you're right, it's just an useless feature. I'll remove it.